.pre-commit-config.yaml

```diff
@@ -34,7 +34,7 @@ repos:
     - id: upgrade-type-hints
       args: [ '--futures=true' ]
   - repo: https://github.com/psf/black
-    rev: 22.1.0
+    rev: 22.3.0
```

Comment:
Don't know why the friendly pre-commit bot hasn't done it yet, but without this, we get errors during the black run (see some of the commits in the middle)

Reply:
Seems like someone wants to mock me with this PR... today the pre-commit fix arrived...

Reply:
this is probably caused by the pre-commit update setting 🤖 
https://github.com/BAMWelDX/weldx/blob/af9df21e2efe10613847cb1d0c6f4b122bfbc880/.pre-commit-config.yaml#L8